### ISSUE TITLE
VoiceOver gets stuck in header

### ISSUE THREAD

**Issue by ryelle** [WordPress/wporg-mu-plugins#29]
[can be addressed post-sotw]

I'm not sure if this is user-error or an issue with the theme, but if I turn on VoiceOver in Safari, I can't get to the rest of the page - I'm stuck in the header & nav menu. Even if I click to focus elsewhere, navigating just takes me right back to the menu. Front page and posts pages. Using the rotor to list out landmarks, links, etc also only shows the header nav.

**Comment by ryelle** [comment]
Looks like this is fixed if I switch the `overlayMenu` attribute to `never` in the navigation blocks. Likely an upstream issue in GB.

**Comment by ryelle** [comment]
🎉  This is fixed with https://github.com/WordPress/gutenberg/pull/37434, which should be out with GB 12.3. I'll come back to close this once that's released and I do another quick test on the staging site.

**Comment by ryelle** [comment]
Confirmed fixed in GB 12.3 & VoiceOver can read the content on `/news-test/` 👍🏻
